src/Components/WebAssembly/WebAssembly.Authentication/src/Interop/AuthenticationService.ts

```diff
@@ -63,12 +63,32 @@ export interface AuthorizeService {
 
 class OidcAuthorizeService implements AuthorizeService {
     private _userManager: UserManager;
-
+    private _intialSilentSignIn: Promise<void> | undefined;
     constructor(userManager: UserManager) {
         this._userManager = userManager;
     }
 
+    async trySilentSignIn() {
+        if (!this._intialSilentSignIn) {
+            this._intialSilentSignIn = (async () => {
+                try {
+                    await this._userManager.signinSilent();
+                    return;
+                } catch (e) {
+                }
```

Comment:
Should we be logging something here maybe? I'm not sure under what circumatances we'd have an exception, but maybe this could make debugging harder.

Reply:
I added a comment, but in essence the answer is that it can fail if the user is not logged in on the underlying IdP.

Reply:
It's unfortunate that there's isn't a `trySigninSilent` to distinguish legit sign-in failures from system exceptions. But if this is beyond our control, no problem.

Reply:
We can check the error message, but there is no point in it, since this is an optimistic optimization. It is not going to mask any error since if you have a system exception it will show up when you try to log in the regular way.